foundationdb/src/tuple.rs

```diff
@@ -431,10 +435,10 @@ impl Single for i64 {
 
 pub trait Tuple: Sized {
     fn encode<W: Write>(&self, _w: &mut W) -> std::io::Result<()>;
-    fn encode_to_vec(&self) -> std::io::Result<Vec<u8>> {
+    fn encode_to_vec(&self) -> Vec<u8> {
         let mut v = Vec::new();
-        self.encode(&mut v)?;
-        Ok(v)
+        self.encode(&mut v).unwrap();
```

Comment:
Can we change this to expect with a message like, “tuple encoding should never fail”?